test/passing/tests/repl.mli

```diff
@@ -0,0 +1,99 @@
+type t = k
+(** VALID BLOCKS:
+
+    Block delimiters should be on their own line:
+    {[ let x = 1 ]}
+
+    As of odoc 2.1, a block can carry metadata:
+    {@ocaml[
+      let x = 2
+    ]}
+
+    An OCaml block that should break:
+    {[
+      let x = 2 in x + x
+    ]}
+
+    A toplevel phrase with no output:
+    {[
+    # let x = 2 and y = 3 in x+y;;
+    ]}
+
+    A toplevel phrase with output:
+    {[
+    # let x = 2;;
+    val x : int = 2
+    ]}
+
+    Many toplevel phrases without output:
+    {[
+    # let x = 2;;
+    # x + 2;;
+    # let x = 2 and y = 3 in x+y;;
+    ]}
+
+    Many toplevel phrases with output:
+    {[
+    # let x = 2;;
+    val x : int = 2
+    # x + 2;;
+    - : int = 4
+    # let x = 2 and y = 3 in x+y;;
+    ]}
+
+    Output are printed afer a newline:
+    {[
+    # let x = 2;; val x : int = 2
```

Comment:
I think it would be fair if ocamlformat didn't handle this as MDX won't produce such blocks and simply left output untouched but putting on its own line is fine as well, whatever is easier for you!